pypose/module/LQR.py

```diff
@@ -0,0 +1,235 @@
+import torch as torch
+import time
+import torch.nn as nn
+
+
+class DP_LQR(nn.Module):
+    r'''
+    Discrete-time finite-horizon Linear Quadratic Regulator (LQR)
+
+    Args:
+        n_state (int): Parameter that determines the dimension of the state.
```

Comment:
n_state, n_ctrl are not necessary? I think we should add these as system properties?